src/web_app/src/pages/ga_hyperParams.tsx

```diff
@@ -163,13 +175,13 @@ export function GaHyperParamsPage() {
                     </Grid>
                     <Grid item xs={6}>
                         <TextField id="filled-basic" variant="filled"
-                            defaultValue={param.params.seed}
+                            defaultValue={seed}
                             onChange={e => setSeed(parseInt(e.target.value))}>
                         </TextField>
                     </Grid>
                     <Grid item xs={6}>
                         <TextField id="filled-basic" variant="filled"
-                            defaultValue={param.params.mutation_rate}
+                            defaultValue={mutation_rate}
                             onChange={e => setMutation_rate(parseInt(e.target.value))}>
```

Comment:
Mutation rate is a float not an Int this will cause problems ...

Reply:
All numbers in TypeScript are either floating point values or integers.

Reply:
Yes, but by using `parseInt` here, you are transforming all float values between 0 and 1 to either 0 or 1. This is quite a big problem for a mutation rate ...

Reply:
Ok I didn't understand the problem, I wil change parseInt to parseFloat.